students/ptfbanks/Lesson02/FizzBuzz.py

```diff
@@ -0,0 +1,20 @@
+#Fixx, Buzz no Bang!
+print("This routine replaces numbers in a range (1 to n, you select n)")
+print ("which are divisible by values (v1, v2) you select.")
+print (" call this by entering fibu (v1, v2, n)")
+
+def fibu (a, b, n):
```

Comment:
it is best to avoid using anonymous one letter vars (there are exceptions).

better for readability: `def fizzbuzz(num1, num2, limit)`